### PR TITLE
Update groups-naming-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information to what triggers 365groups naming policy

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3728

File: microsoft-365/solutions/groups-naming-policy.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ description: "Learn how to create a naming policy for Microsoft 365 groups."
 
 You can use a group naming policy to enforce a consistent naming strategy for groups created by users in your organization. A naming policy can help you and your users identify the function of the group, membership, geographic region, or who created the group. The naming policy can also help categorize groups in the address book. You can use the policy to block specific words from being used in group names and aliases.
 
-The naming policy is applied to groups that are created across all groups workloads (like Outlook, Microsoft Teams, SharePoint, Planner, Yammer, etc.). It gets applied to both the group name and group alias. It gets applied when a user creates a group and when group name or alias is edited for an existing group.
+The naming policy is applied to groups that are created across all groups workloads (like Outlook, Microsoft Teams, SharePoint, Planner, Yammer, etc.). It gets applied to both the group name and group alias. It also gets applied when a user creates a group and when group name, alias, description or avatar is edited for an existing group. Even clicking on the update button triggers the naming policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/groups-naming-policy.md]:
```suggestion
The naming policy is applied to groups that are created across all groups workloads (like Outlook, Microsoft Teams, SharePoint, Planner, Yammer, etc.). It gets applied to both the group name and group alias. It also gets applied when a user creates a group and when the group name, alias, description, or avatar is edited for an existing group. Even clicking on the update button triggers the naming policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you [review this update](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/3803/files)?

**Review by ArunMSFT — CHANGES_REQUESTED** [review]
The first of the updates look good but I'm not clear what this second update is about and what additional value it adds.

`Even clicking on the update button triggers the naming policy.`

**Comment by msbemba** [comment]
@ArunMSFT  I meant updating the naming policy. I see the ambiguity there, So I'll remove that part from the PR.

**Pull request merged**